keyboards/kprepublic/cstc40/keymaps/via/rules.mk

```diff
@@ -0,0 +1,2 @@
+TRI_LAYER_ENABLE = yes
```

Comment:
Via enables this by default, so it's not necessary, here. 
```suggestion
```

Reply:
Thank you for the information.